src/west/app/project.py

```diff
@@ -401,9 +401,24 @@ def do_add_parser(self, parser_adder):
             - cloned: "cloned" if the project has been cloned, "not-cloned"
               otherwise
             - clone_depth: project clone depth if specified, "None" otherwise
+            - groups: project groups, as a comma-separated list
+
+            INCLUDED PROJECTS
+            -----------------
+
+            By default, output does not include information for any project
+            that "west update" would not update by default due to:
+
+            - the "groups" / "no-groups" manifest file keys
+            - the "manifest.groups" / "manifest.no-groups" local configuration
+              options in .west/config
+
+            To override this, give an explicit list of projects (by name or
+            path), or use "west list --all".
```

Comment:
Guessing that some users might come here to understand why `project-off` is not printed with `west list`, then maybe we should give a small hint, like:
> Additional groups can be added to manifest.groups / manifest.no-groups using west config.

Reply:
It's right there in the bullet list, isn't it?

```
            - the "manifest.groups" / "manifest.no-groups" local configuration
              options in .west/config
```

Reply:
yes, I did notice, I was more thinking that a reference to `west config` would be nice, so that users would be hinted on how to update `.west/config`.

The bullet only talks about `.west/config`, thus assuming the user already has knowledge about `west config`.

Reply:
There's only so much we can do in the --help output. We have pretty good documentation on configuration linked to right from the front page of the west docs:

https://docs.zephyrproject.org/latest/guides/west/index.html

Do we really need to repeat it all here?

Reply:
not necessarily